server/lib/set-headers-to-preload-assets.js

```diff
@@ -70,7 +70,7 @@ function setHeadersToPreloadAssets(res, pageOptions) {
     return `<${scriptUrl}>; rel=modulepreload`;
   });
 
-  res.append('Link', [].concat(styleLinks, fontLinks, imageLinks, scriptLinks).join(', '));
+  res.set('Link', [].concat(styleLinks, fontLinks, imageLinks, scriptLinks).join(', '));
```

Comment:
Not sure why this was `res.append(...)` before. Co-pilot 🤷

Reply:
Perhaps we should keep it as `res.append(...)` as it's completely valid to do, see https://expressjs.com/en/api.html#res.append

Reply:
🤷 We don't need the `append` functionality and only expect to set this header in one place. Will leave it now as it's easier to `Ctrl + F for `res.set(...)` everywhere